src/main/webapp/script.js

```diff
@@ -65,11 +65,12 @@ function displayPotentialMatchInfo(pmID) {
     let carouselContainer = document.getElementById("carousel-inner");
     let numPhotos = 5; //hard coded value for now, once blobstore works, this will be the amount of images the user has uploaded
     for (let i = 0; i < numPhotos; i++) {
+      let slideshowElement;
```

Comment:
For reference only - Use of `let` here is good because you are not assigning at instantiation time, but later.